timesketch/lib/datastores/opensearch.py

```diff
@@ -1237,3 +1247,290 @@ def flush_queued_events(self, retry_count=0):
 
         self.import_events = []
         return return_dict
+
+    def _export_slice_worker_pit(
+        self,
+        slice_id: int,
+        num_slices: int,
+        index_list: List[str],
+        query_body: Dict[str, Any],
+        current_sort_criteria: List[Dict[str, Any]],
+        current_page_size: int,
+        current_pit_keep_alive: str,
+        output_queue: queue.Queue,
+        stop_event: threading.Event,
+        worker_request_timeout: int
+    ):
+        """
+        Worker function for a single slice in a PIT export.
+        Fetches documents for its assigned slice and puts them onto the output_queue.
```

Comment:
could we make the docstring a bit more descriptive on what is going on, when errors are dropped, how the parameters are used?

Reply:
done